VelocidiSDK/VelocidiSDKTests/VelocidiTests.swift

```diff
@@ -61,20 +62,37 @@ class NetworkTests: QuickSpec {
                         return request.url!.absoluteString.starts(with: trackURL)
                     }, responseBuilder(url:trackURL, expectedData:trackingEvent.toJSONData()))
 
-                    var success: Bool? = nil
 
                     let config = VSDKConfig(trackingBaseUrl: trackURL, matchURL)
                     VSDKVelocidi.start(config!)
 
                     VSDKVelocidi.sharedInstance().track(trackingEvent, onSuccess:{ (response: URLResponse, responseObject: Any) in
-                        success = true
-                    }, onFailure:{(error: Error) in
-                        NSLog("Error \(error.localizedDescription)")
-                        success = false
+                        success.pointee = true
+                    }, onFailure:{(err: Error) in
+                        success.pointee = false
+                        error.pointee = err
                     })
-
-                    expect(success).toEventuallyNot(beNil(), timeout: 4)
-                    expect(success).toEventuallyNot(beFalse())
+                }
+                
+                if #available(iOS 14, *) { // ios14 is opt-in
+                    it("should fail to execute tracking requests when the user is not opted-in in iOS 14") {
```

Comment:
Can we also test that tracking requests are correctly performed when the user opts-in in iOS 14?

Reply:
I don't think we have a way to do that without injecting another dependency in `VSDKVelocidi`.

Reply:
Also added 99872d4